nemo/collections/asr/data/audio_to_text_dataset.py

```diff
@@ -218,6 +218,42 @@ def get_tarred_dataset(
 
         datasets.append(dataset)
 
+    if len(datasets) > 1:
+        if config.get('bucketing_batch_size', None) is not None and config['bucketing_batch_size'] > 0:
+            if config['batch_size'] != 1:
+                raise ValueError(
+                    f"batch_size should be set to one when bucketing_batch_size is set and adaptive bucketing is enabled (batch_size={config['batch_size']}!"
+                )
+
+            if type(config['bucketing_batch_size']) == int:
```

Comment:
Pull this logic which determines bucketing batch sizes into another method and call it here. Makes it easier to modularize in the future.

Reply:
moved it.